### DISCUSSION TITLE
Typing text field with "contentEditable" element only fill 1 letter?

### DISCUSSION THREAD

**Discussion by kolampixel** [AutomaApp/automa#1049] [General]
Hi,

I used "form" to fill in the DIV elements with the "contentEditable" attribute, but why only fill in 1(st) letter?

![2023-01-02 18_39_06-Acara _ Facebook](https://user-images.githubusercontent.com/121801559/210227472-97537ab6-5a6d-4bf5-b5c9-08069215303c.png)

![2023-01-02 18_39_51-contenteditable workflow - Automa](https://user-images.githubusercontent.com/121801559/210227570-be3a01f3-37b2-4825-9feb-473e03e71c8d.png)

Anyone have the same case? Thankss

**Comment by Kholid060** [comment]
Open the block settings for the Forms block and enable the "Debug mode" option
![image](https://user-images.githubusercontent.com/22908993/210292354-bc5920ae-4be0-4c7f-be90-b7b30b357414.png)

**Reply by kolampixel** [reply]
Thank you saya coba mas.

SOLVED.
